clinica/pipelines/statistics_volume/statistics_volume_cli.py

```diff
@@ -0,0 +1,112 @@
+
+        optional.add_argument("--custom_files", "-cf", type=str, default=None,
+                              help=('Custom file string. Specify filename using * when the subject or session name appear. '
+                                    + 'Example : \'*_task-rest_acq-fdg_pet_space-Ixi549Space_pet.nii.gz\' will grab the'
+                                    + ' corresponding file in all the subjects/sessions'))
+
+        optional.add_argument("-tup", "--threshold_uncorrected_pvalue",
+                              type=float, default=0.001,
+                              help='Threshold to display the uncorrected p-value '
+                                   + '(--threshold_uncorrected_pvalue 0.001).')
+
+        optional.add_argument("-tcp", "--threshold_corrected_pvalue",
+                              type=float, default=0.05,
+                              help='Threshold to display the corrected p-value '
+                                   '(default: --threshold_corrected_pvalue 0.05)')
+
+        optional.add_argument("-gic", "--group_id_caps", type=str, default=None,
+                              help='Name of the group that Clinica needs to use to grab input file')
+
+        optional.add_argument("-fwhm", "--smoothing", type=int, default=8,
+                              help='Full Width at Half Maximum (FWHM) of the smoothing used in your input file.')
+
+        # Clinica standard arguments (e.g. --n_procs)
+        self.add_clinica_standard_arguments()
+
+    def run_command(self, args):
+        import os
+        from networkx import Graph
+        from colorama import Fore
+        from .statistics_volume_pipeline import StatisticsVolume
+        from clinica.utils.stream import cprint
+        from clinica.utils.ux import print_end_pipeline, print_crash_files_and_exit
+
+        # Most of the time, you will want to instantiate your pipeline with a
+        # BIDS and/or CAPS directory as inputs. If the BIDS directory is not needed
+        # for your pipeline, simply remove:
+        # bids_directory=self.absolute_path(args.bids_directory),
+        pipeline = StatisticsVolume(
+            caps_directory=self.absolute_path(args.caps_directory),
+            tsv_file=self.absolute_path(args.subject_visits_with_covariates_tsv),
+            base_dir=self.absolute_path(args.working_directory),
+            name=self.name
+        )
+        pipeline.parameters = {'contrast': args.contrast,
+                               'feature_type': args.feature_type,
+                               'group_id': args.group_id,
+                               'custom_files': args.custom_files,
+                               'threshold_uncorrected_pvalue': args.threshold_uncorrected_pvalue,
+                               'threshold_corrected_pvalue': args.threshold_corrected_pvalue,
+                               'group_id_caps': args.group_id_caps,
+                               'smoothing': args.smoothing}
```

Comment:
Since PR #51, parameters are set before instancitation of the pipeline. This should look like:
```python
        parameters = {
            'contrast': args.contrast,
            'feature_type': args.feature_type,
            'group_id': args.group_id,
            'custom_files': args.custom_files,
            'threshold_uncorrected_pvalue': args.threshold_uncorrected_pvalue,
            'threshold_corrected_pvalue': args.threshold_corrected_pvalue,
            'group_id_caps': args.group_id_caps,
            'smoothing': args.smoothing
        }
        pipeline = StatisticsVolume(
            caps_directory=self.absolute_path(args.caps_directory),
            tsv_file=self.absolute_path(args.subject_visits_with_covariates_tsv),
            base_dir=self.absolute_path(args.working_directory),
            parameters=parameters,
            name=self.name
        )
```

Reply:
Agreed